### PR TITLE
Nodes: Bleach Node

### PR DESCRIPTION
**Pull request by cmhhelgeson**
Related issue: 

**Description**

Port BleachBypassShader to Node system: 

```typescript
const scenePass = pixelationPass( scene, camera, effectController.pixelSize, effectController.normalEdgeStrength, effectController.depthEdgeStrength );
const bleachPass = bleach( { color: scenePass.renderOutput(), opacity: effectController.bleachMix } );
postProcessing.outputNode = bleachPass;
```

[Live WebGPU Example](https://raw.githack.com/cmhhelgeson/three.js/webgpu_post_test/examples/webgpu_postprocessing_pixel.html)

[Live WebGL Comparison Example](https://raw.githack.com/cmhhelgeson/three.js/webgpu_post_test/examples/webgl_postprocessing_pixel.html)

### CODE REVIEW

**Comment by Mugen87** [comment]
> const bleachPass = bleach( { color: scenePass.renderOutput(), opacity: effectController.bleachMix } );

Please write the code in the examples like so:
```js
const bleachPass = bleach( scenePass.renderOutput(),  effectController.bleachMix );
```

File: src/nodes/display/BleachBypassNode.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { addNodeElement, float, tslFn, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { min, max, dot, mix } from '../math/MathNode.js';
+
+import { property } from '../core/PropertyNode.js';
+
+export const bleach = /*@__PURE__*/ tslFn( ( { color, opacity = 1.0 } ) => {
+
+	const base = property( 'vec4', 'base' ).assign( color );
```
Comment (sunag) [inline src/nodes/display/BleachBypassNode.js:8]: I think `property` and `assign` can be avoided here?

**Comment by cmhhelgeson** [comment]
> > const bleachPass = bleach( { color: scenePass.renderOutput(), opacity: effectController.bleachMix } );
> 
> Please make sure the code can be written like so:
> 
> ```js
> const bleachPass = bleach( scenePass.renderOutput(), effectController.bleachMix );
> ```
> 
> You possibly have to update `sepia()` as well.

Just tested, both work with .renderOutput() with example already using it as well.

File: src/nodes/display/BleachBypassNode.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { addNodeElement, float, tslFn, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { min, max, dot, mix } from '../math/MathNode.js';
+
+export const bleach = /*@__PURE__*/ tslFn( ( [ color, opacity = 1 ] ) => {
```
Comment (Mugen87) [inline src/nodes/display/BleachBypassNode.js:4]: Please do not use `{}` for the parameters. If you do so, the pass does not work when doing this:
```js
const scenePass = pass( scene, camera );
const scenePassColor = scenePass.getTextureNode();

postProcessing.outputNode = scenePassColor.bleach();
```

File: src/nodes/display/BleachBypassNode.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { addNodeElement, float, tslFn, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { min, max, mix } from '../math/MathNode.js';
+import { luminance } from './ColorAdjustmentNode.js';
+
+export const bleach = /*@__PURE__*/ tslFn( ( [ color, opacity = 1 ] ) => {
+
+	const base = color;
+	const lum = luminance( base.rgb );
```
Comment (Mugen87) [inline src/nodes/display/BleachBypassNode.js:8]: I think it's more correct to use `luminance()` here instead of using a custom computation.

Comment (Mugen87) [reply src/nodes/display/BleachBypassNode.js:8]: @donmccurdy It seems there is another place in the code where `luminance()` could be used. 

https://github.com/mrdoob/three.js/blob/93da17b807b8f0dd0cfabd0b45f8d4d8b2b6c7a4/examples/jsm/shaders/BleachBypassShader.js#L41-L43

The coefficients `0.25, 0.65, 0.1` seems inconsistent to me...

Comment (donmccurdy) [reply src/nodes/display/BleachBypassNode.js:8]: Agreed! I suspect they're just loose approximations, maybe the original implementation didn't have a particular color space in mind. We can use the default luminance coefficients instead (as you've done).

**Review by Mugen87 — APPROVED** [review]

**Pull request merged**